container/src/services/webcomponents.service.ts

```diff
@@ -100,6 +100,29 @@ export class WebComponentService {
             const options = { fromContext, fromClosestContext, fromVirtualTreeRoot, fromParent, nodeParams, ...settings };
             this.dispatchLuigiEvent(Events.NAVIGATION_REQUEST, { link: route , ...options});
           },
+          navigateToIntent: (semanticSlug: string, params = {}, settings = {}): void => {

```

Comment:
I think you don't need the settings parameter here

`navigateToIntent: (semanticSlug: string, params = {}): void => {`

Can keep it like this, in sync with the rest. The **linkManagerInstance.navigate(newPath)** could be called without needing settings.